### ISSUE TITLE
Add option to disable some node from UI

### ISSUE THREAD

**Issue by gre** [auchenberg/dependo#12]
Hi guys,

cheers for this awesome tool. That's a nice start and I would like to see more things in the UI.

I like the D3 graph controls, however I would like to easily "disable" some nodes to visualise more easily my dependencies.

For instance: http://bl.ocks.org/gre/raw/8caa85c90b134f3c8852/ is the graph of my app,
but it is barely readable because basically everyone depends on "react" , "lodash" and "q". I don't want to statically remove those nodes because that's the truth, I just want to disable (delete/or grey out) from the UI a node and all its connection.

Thanks

**Comment by trusktr** [comment]
Yeah, perhaps single clicking on a line or node makes it permanently greyed out (rather than on hover of other node), and clicking on it again makes it back to the normal (not greyed out, greys out when hovering on another node).
